src/dgrid/DgridInnerWrapper.ts

```diff
@@ -0,0 +1,216 @@
+import { dom } from '@dojo/framework/widget-core/d';
```

Comment:
Can this whole file just not be in the same module as the `DgridWrapper`, don't really want people to use this externally.